sources/packages/api/src/route-controllers/application/models/application.model.ts

```diff
@@ -269,3 +271,11 @@ export const transformToApplicationDetailDto = (
     programYearEndDate: applicationDetail.programYear.endDate,
   };
 };
+
+/**
+ * Enum for COE enrollment period.
+ */
+export enum EnrollmentPeriod {
```

Comment:
Does this enum really belongs to application? It seems more on COE scope, doesn't?

Reply:
Moved enum to .model in disbursement service as discussed.